crates/ra_hir_expand/src/builtin_macro.rs

```diff
@@ -171,3 +179,23 @@ fn file_expand(
 
     Ok(expanded)
 }
+
+fn compile_error_expand(
+    _db: &dyn AstDatabase,
+    _id: MacroCallId,
+    tt: &tt::Subtree,
+) -> Result<tt::Subtree, mbe::ExpandError> {
+    if tt.count() == 1 {
+        match &tt.token_trees[0] {
+            tt::TokenTree::Leaf(tt::Leaf::Literal(it)) => {
+                let s = it.text.as_str();
+                if s.starts_with(r#"""#) && s.ends_with(r#"""#) {
```

Comment:
It may be a raw string, which start with `#` , just checking whether it contains a double quote will be enough.